README.md

```diff
@@ -6,4 +6,4 @@ Files were created using Replit editor, then downloded to personal computer and
 Project is a requirment of Microverse mini bootcamp, and specifications are indicated in this link:
 https://github.com/TheOdinProject/curriculum/blob/b5b6fc5cfedbcc28005eb92aca8ba79e0e81bdc8/html_css/project_media.md
 
-Change
\ No newline at end of file
+Change 2
```

Comment:
Please delete this comments

Reply:
Please tray to have your README file to follow guidelines of Microverse template

Reply:
All sections of the template must be present in your README file, event if some sections dosn't apply to your project. Also keep format of section as the template.